lmp/customize-target

```diff
@@ -48,6 +55,9 @@ def merge(args):
     updates = []
     for idx, (tgt_tag, apps_tag) in enumerate(tagmgr.tags):
         tgt = targets[args.target_name]
+        tgt['custom']['lmp-manifest-sha'] = git_hash('/srv/oe/.repo/manifests')
+        tgt['custom']['meta-subscriber-overrides-sha'] = git_hash(
+            '/srv/oe/layers/meta-subscriber-overrides')
```

Comment:
is is also going to be called for the public builds or just for subscribers?

Reply:
good catch! force push fixed